packages/turf-point-on-line/index.js

```diff
@@ -1,18 +1,21 @@
-var distance = require('@turf/distance');
+var meta = require('@turf/meta');
 var helpers = require('@turf/helpers');
 var bearing = require('@turf/bearing');
+var distance = require('@turf/distance');
 var destination = require('@turf/destination');
 var lineIntersects = require('@turf/line-intersect');
 var point = helpers.point;
+var coordAll = meta.coordAll;
 var lineString = helpers.lineString;
+
 /**
- * Takes a {@link Point} and a {@link LineString} and calculates the closest Point on the LineString.
+ * Takes a {@link Point} and a {@link LineString} and calculates the closest Point on the (Multi)LineString.
  *
  * @name pointOnLine
- * @param {Feature<LineString>} line line to snap to
+ * @param {Feature<LineString>|Feature<MultiLineString>} lines lines to snap to
```

Comment:
To support both Feature & Geometry, so far the JSDocs have been declared like this:

```js
/**
 * @param {Geometry|Feature<LineString|MultiLineString>} lines lines to snap to
 * @param {Geometry|Feature<Point>|number[]} pt point to snap from
```